reconcile/utils/ocm.py

```diff
@@ -955,6 +991,7 @@ def __init__(
         settings=None,
         init_provision_shards=False,
         init_addons=False,
+        init_version_gates=False,
```

Comment:
why is the default different between `OCMMap` and `OCM` ?

Reply:
Fixed